src/main/java/nextstep/courses/domain/lectures/LectureRecruitingStatus.java

```diff
@@ -0,0 +1,18 @@
+package nextstep.courses.domain.lectures;
+
+public enum LectureRecruitingStatus {
+  RECRUITING("RECRUITING")
+  , CLOSING("CLOSING")
+  , PREPARING("PREPARING")
+  , NO_MATCH("NO_MATCH")
+  ;
```

Comment:
```suggestion
  RECRUITING("RECRUITING"),
  CLOSING("CLOSING"),
  PREPARING("PREPARING"),
  NO_MATCH("NO_MATCH");
```
위와 같이 스타일로 구현하는 것도 가능함